modules/chanserv/set_gameserv.c

```diff
@@ -31,6 +31,12 @@ cs_cmd_set_gameserv(struct sourceinfo *si, int parc, char *parv[])
 		return;
 	}
 
+	if (metadata_find(mc, "private:close:closer"))
+	{
+		command_fail(si, fault_noprivs, STR_CHANNEL_IS_CLOSED, parv[0]);
+		return;
+	}
+
```

Comment:
This needs to be after the block below it; permissions checks should come before closure checks.